docs/3.x/tech/tutorials/exporting-lists.adoc

```diff
@@ -1,35 +1,35 @@
 ---
 layout: doc
-title: Exporting Lists and Configurations
-description: Learn how to export your lists and Configurations in CleanSpeak
+title: Exporting Filter Lists Configuration
```

Comment:
Not sure if it is the screenshots or some other style, but each of the screenshots is a different width.